src/data/zwave_js.ts

```diff
@@ -388,11 +389,9 @@ export const enum NodeStatus {
 export interface ZwaveJSProvisioningEntry {
   /** The device specific key (DSK) in the form aaaaa-bbbbb-ccccc-ddddd-eeeee-fffff-11111-22222 */
   dsk: string;
-  security_classes: SecurityClass[];
-  additional_properties: {
-    nodeId?: number;
-    [prop: string]: any;
-  };
+  securityClasses: SecurityClass[];
+  nodeId?: number;
+  [prop: string]: any;
```

Comment:
Isn't this still under `additionalProperties`? https://github.com/home-assistant/core/pull/117400/files#diff-06eb7dbc11f1df244d8fa45fe6751883b5e723b0c56e90c2f1e6cec980fd6c67R207

![CleanShot 2024-05-14 at 09 22 08@2x](https://github.com/home-assistant/frontend/assets/1444314/1adc9b88-cb14-43b8-8aa6-ffd74d014978)

Reply:
no, this type is for the payload sent from the frontend to the backend via WS, and here we are converting the data to `QRProvisioningInformation`. This is consistent with what we get from the server, we have `additional_properties` because that was the only way I could think of to support additional key value pairs that we didn't knwo in advance